docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -89,6 +106,24 @@ Install the k8ssandra-cluster chart with the following properties. Backup and re
 
 `helm install k8ssandra-cluster-1 k8ssandra/k8ssandra-cluster --set backupRestore.medusa.bucketName=k8ssanda-bucket-dev --set backupRestore.medusa.bucketSecret=medusa-bucket-secret`
 
+As an alternative to specifying those --set parameters, you can use the provided [backup-restore-values.yaml](./backup-restore-values.yaml) file. It contains:
```

Comment:
I think it would be better to remove the example of setting all the properties on the command line. In reality, I suspect it most often be done with a file that can be stored in version control.

Reply:
I agree - done in next commit.